### ISSUE TITLE
v2.6.1.2 does not contain MAlonzo/RTE/Float.hs

### ISSUE THREAD

**Issue by rintcius** [agda/agda#5146]
I was trying out hello world on v.2.6.1.2 and ran into https://github.com/agda/agda/issues/3619.

Maybe I'm missing something, but from the milestone mentioned in https://github.com/agda/agda/issues/3619#issuecomment-699661364 I think https://github.com/agda/agda/commit/aa17ef039db11c11709b0cef357691c31e78d991 should be included in v.2.6.1.2?

```
$ git co master
Switched to branch 'master'
$ ls -R src/data/MAlonzo/src/MAlonzo/
RTE    RTE.hs

src/data/MAlonzo/src/MAlonzo//RTE:
Float.hs

$ git co v2.6.1.2
Note: switching to 'v2.6.1.2'.
$ ls -R src/data/MAlonzo/src/MAlonzo/
RTE.hs
```

**Comment by andreasabel** [comment]
Thanks for reporting!

v2.6.1.2 does not fix #3619.  The milestone 2.6.1.2 is incorrect at #3619.

**Comment by andreasabel** [comment]
Will be released in 2.6.2.

**Comment by andreasabel** [comment]
That should stay on the 2.6.2 milestone, as we will release the fix there.
